qemu/tests/balloon_stress.py

```diff
@@ -5,31 +5,34 @@
 from virttest import utils_test
 from virttest import error_context
 from qemu.tests.balloon_check import BallooningTestWin
+from qemu.tests.balloon_check import BallooningTestLinux
 
 
 @error_context.context_aware
 def run(test, params, env):
     """
     Qemu balloon device stress test:
     1) boot guest with balloon device
-    2) enable driver verifier in guest
-    3) reboot guest (optional)
-    4) check device using right driver in guest.
-    5) play video in background repeatly
-    6) balloon memory in monitor in loop
+    2) enable driver verifier in guest (Windows only)
+    3) run stress in background repeatly
+    4) balloon memory in monitor in loop
 
     :param test: QEMU test object
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
 
-    def run_video():
+    def run_stress(vm):
         """
-        Run video in background
+        Run stress in background
         """
         while True:
-            utils_test.run_virt_sub_test(test, params, env,
-                                         params.get("video_test"))
+            if params['os_type'] == 'windows':
+                utils_test.run_virt_sub_test(test, params, env,
+                                             params.get("stress_test"))
```

Comment:
You forgot to change the param in the following place.
`qemu/tests/cfg/driver_load_stress.cfg`

Reply:
Modified, Thanks!